recipes/persistent.rb

```diff
@@ -14,9 +14,10 @@
 
 devices = Dir.glob('/dev/xvd*')
 if devices.empty?
-  next_mount = "a"
```

Comment:
i'm pretty sure i got failures until i used g as the starting letter due to some aws api error response